### PR TITLE
40 - Missions Next Steps

### PR DESCRIPTION
**Pull request by graceschae**
Screenshot 2023-07-19 at 5 36 27 PM

### CODE REVIEW

File: frontend/src/pages/next-steps/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import * as React from "react";
 
 import Layout from "../../components/layout";
 import Seo from "../../components/seo";
+import MissionsNextStep from "../../components/page-next-steps/missionsNextSteps";
 
 const NextStepsPage = () => (
   <Layout>
```
Comment (thedomcab) [inline frontend/src/pages/next-steps/index.js:8]:
```suggestion
  <Layout>
     <div className="max-w-container mx-auto">
```

File: frontend/src/pages/next-steps/index.js [hunk 2]
```diff
@@ -10,7 +11,7 @@ const NextStepsPage = () => (
     <div>Transformation classes section</div>
     <div>Baptism section</div>
     <div>Ministry teams section</div>
-    <div>Missions section</div>
+    <MissionsNextStep />
   </Layout>
```
Comment (thedomcab) [inline frontend/src/pages/next-steps/index.js:15]:
```suggestion
</div>
  </Layout>
```

**Review by thedomcab — COMMENTED** [review]
just make a minor change and should be good.

**Review by xumaple — COMMENTED** [review]
Sorry for not posting about this earlier, but the missions section was slightly altered for our MVP to make things easier (in this case, removing the link and replacing it with just telling people to send an email)

I'll post more details on teams

**Review by thedomcab — DISMISSED** [review]

**Comment by graceschae** [comment]
Screenshot 2023-07-20 at 5 55 13 PM
Updated view

**Review by thedomcab — APPROVED** [review]

**Pull request merged**